sapi/fpm/tests/cpuaffinity-range.phpt

```diff
@@ -0,0 +1,55 @@
+--TEST--
+FPM: cpu affinity test, range
+--SKIPIF--
+<?php include "skipif.inc";
+
+if (!str_contains(PHP_OS, 'Linux') && !str_contains(PHP_OS, 'FreeBSD')) {
+    die('skipped supported only on Linux and FreeBSD');
+}
+
+if (str_contains(PHP_OS, 'Linux')) {
+    $cmd = 'nproc';
+} else {
+    $cmd = 'sysctl hw.ncpus';
+}
+
+$nproc = intval(exec($cmd));
+if ($nproc < 4) {
+    die('skipped supported only on 4 cores or more environments');
+}
```

Comment:
Can you integrate those independently to the Tester (static methods) as done elsewhere...

Reply:
good point.